### ISSUE TITLE
Seems to be blocking several keyboard shortcuts.

### ISSUE THREAD

**Issue by hooloovooo** [gaearon/redux-devtools-dock-monitor#9]
When i try using redux-devtools-dock-monitor it seems to be blocking all keyboard shortcuts using ctrl in them, so i cannot use ctrl + c or ctrl + r or a page which uses this. When commenting out the handleKeyDown call it all seems to work again.

**Comment by loganetherton** [comment]
Yes, I found this to be the case as well. Simply updating `DockMonitor.prototype.handleKeyDown` to only prevent default on the targeted key seems to do the trick.

```
DockMonitor.prototype.handleKeyDown = function handleKeyDown(e) {
    if (!e.ctrlKey) {
      return;
    }
    //e.preventDefault(); -- Original

    var key = event.keyCode || event.which;
    var char = String.fromCharCode(key);
    switch (char.toUpperCase()) {
      case this.props.toggleVisibilityKey.toUpperCase():
        e.preventDefault(); // Prevent only the targeted key event
        this.props.dispatch(_actions.toggleVisibility());
        break;
      case this.props.changePositionKey.toUpperCase():
        e.preventDefault();  // Prevent only the targeted key event
        this.props.dispatch(_actions.changePosition());
        break;
      default:
        break;
    }
  };
```

I do have to say, I haven't explored the ramifications of this change, but I can't imagine how it would hurt. Seemed kind of small to submit a PR for.

**Comment by gaearon** [comment]
This is fixed in master. https://github.com/gaearon/redux-devtools-dock-monitor/commit/1e6980ae181bb773b0d5451d5316ad1b011789ec
It will be up in the next release.

**Comment by gaearon** [comment]
The fix is released in 1.0.1, requires DevTools 3.0 stable.
